### PR TITLE
feat(ingestion): make image runtime configurable

### PR DESCRIPTION
**Pull request by harshach**
### Describe your changes :

I worked on the ..... because ...

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have performed a self-review of my own. 
- [ ] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: ingestion/ingestion_dependency.sh [hunk 1]
```diff
@@ -10,17 +10,36 @@
...
-(while ! wget -O /dev/null -o /dev/null http://openmetadata-server:8585/api/v1/tables/name/bigquery_gcp.shopify.fact_sale; do sleep 5; done; sleep 9; curl -u admin:admin --data '{"dag_run_id":"sample_dbt_1"}' -H "Content-type: application/json" -X POST http://ingestion:8080/api/v1/dags/sample_dbt/dagRuns) &
-(while ! wget -O /dev/null -o /dev/null http://openmetadata-server:8585/api/v1/tables/name/bigquery_gcp.shopify.fact_sale; do sleep 5; done; sleep 10; curl -u admin:admin --data '{"dag_run_id":"index_metadata_1"}' -H "Content-type: application/json" -X POST http://ingestion:8080/api/v1/dags/index_metadata/dagRuns) &
-airflow standalone
\ No newline at end of file
+
+if [ ! -z "${LOAD_SAMPLE}" ]; then
+    (while ! wget -O /dev/null -o /dev/null http://ingestion:8080; do sleep 5; done; sleep 5; curl -u admin:admin --data '{"dag_run_id":"sample_data_1"}' -H "Content-type: application/json" -X POST http://ingestion:8080/api/v1/dags/sample_data/dagRuns) &
```
Comment (mithmatt) [inline ingestion/ingestion_dependency.sh:39]: There is a reference to admin:admin in each of the curl requests.

Should these be replaced with AIRFLOW_ADMIN_USER and AIRFLOW_ADMIN_PASSWORD

Comment (JorgenEvens) [reply ingestion/ingestion_dependency.sh]: I've fixed that now.

**Comment by harshach** [comment]
Thanks @JorgenEvens . @akash-jain-10 can you check and merge this.

**Comment by akash-jain-10** [comment]
@JorgenEvens  - Can you merge these changes with main branch changes and update the PR please.

**Comment by JorgenEvens** [comment]
@akash-jain-10 The branch has been rebased using current `main` branch.

**Review by vivekratnavel — APPROVED** [review]

**Pull request merged**